### ISSUE TITLE
Funcionando?

### ISSUE THREAD

**Issue by Arthuravila** [ogilvieira/anime-crawler-api#2]
Olá, não está funcionando? Achei interessante para estudo, porém não consegui ver o funcionamento.

**Comment by AdaiasMagdiel** [comment]
Não, não está mais funcionando. Possivelmente a estrutura do site em questão não funciona mais da mesma forma.

**Comment by DioLps** [comment]
Jovens, fiz uma pr corrigindo os problemas: https://github.com/ogilvieira/anime-crawler-api/pull/5

**Comment by Gabslocked** [comment]
Mesmo após a pr, nao consigo puxar a lista de episodeos

**Comment by DioLps** [comment]
> consigo puxar a lista de episodeos

Poxa, você poderia me mandar os passos para repetir o que você fez? a pr ainda não foi aprovada, então não sei ao certo se você deu pull do meu fork ou do projeto dele

**Comment by SamuelGadiel** [comment]
Heroku fora do ar. Mostrando a seguinte mensagem

"Application error
An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command"
